### PR TITLE
final updates

### PR DESCRIPTION
**Pull request by ashishchib2**
Removed unnecessary files and added some features.

### CODE REVIEW

File: src/services/input_blob_handler.py [hunk 1]
```diff
@@ -182,17 +181,6 @@ def move_blob_from_source_folder_to_destination_folder_in_azure_blob_storage(
 
 
 def get_sas_url(blob_path: str, blob_service_client: BlobServiceClient) -> str:
-    """
```
Comment (M-Razdan) [inline src/services/input_blob_handler.py:185]: Removing function descriptions might be counterproductive to code readability.

File: src/services/input_blob_handler.py [hunk 2]
```diff
@@ -212,64 +200,53 @@ def get_sas_url(blob_path: str, blob_service_client: BlobServiceClient) -> str:
 
 def set_processing_status_and_move_completed_blobs(
     blob_service_client: BlobServiceClient, input_blob: InputBlob, is_error: bool
-) -> InputBlob:
-    """
```
Comment (M-Razdan) [inline src/services/input_blob_handler.py:216]: Another removed description. Double check if removing is necessary.

**Review by M-Razdan — COMMENTED** [review]
The code apparently runs as intended so that shouldn't be an issue.
Two potentially important things to note however:

1. Committed changes remove a lot of function descriptions/definitions. That might impact code readability and that too for no benefits. Double check if that is necessary.
2. A lot of exception handling has been modified. Might be prudent to check if exceptions are still handled properly in the production version.

File: src/services/input_blob_handler.py (hunk 2 above)
Comment (ashishchib2) [reply src/services/input_blob_handler.py:216]: the function name is self explanatory, that was the reason I removed  description.

**Pull request merged**